Test/TestRawFileReader.cs

```diff
@@ -47,12 +47,14 @@ public static void TestDynamicConnectionRawFileReader()
 
             ThermoRawFileReaderData.InitiateDynamicConnection(path);
 
+            var msOrders = ThermoRawFileReaderData.GetMsOrdersByScanInDynamicConnection();
```

Comment:
Is there a reason for this? If code coverage, include an assert. If just checking to make sure it doesn't crash, add a comment.